### PR TITLE
i18n(zh-CN): Update `view-transitions.mdx`, `cloudflare.mdx`

### PR DESCRIPTION
**Pull request by Genteure**
#### Description (required)

Updated translation based on source texts at [9049c78](https://github.com/withastro/docs/tree/9049c78bf535b132d706a8b534f040c27958988d).

| File | Source | Source Diff | Other Links |
| --- | --- | --- | --- |
| `guides/deploy/cloudflare.mdx` | [source@`9049c78`](https://github.com/withastro/docs/blob/9049c78bf535b132d706a8b534f040c27958988d/src/content/docs/en/guides/deploy/cloudflare.mdx) | [`57d124a..9049c78`](https://i18n-tools.genteure.com/diff#withastro/docs/57d124a0451093fc8382f024f556dc9821783b7b..9049c78bf535b132d706a8b534f040c27958988d/src/content/docs/en/guides/deploy/cloudflare.mdx) | [blame@`9049c78`](https://github.com/withastro/docs/blame/9049c78bf535b132d706a8b534f040c27958988d/src/content/docs/en/guides/deploy/cloudflare.mdx) <br> [blame@`main`](https://github.com/withastro/docs/blame/main/src/content/docs/en/guides/deploy/cloudflare.mdx) <br> [history@`main`](https://github.com/withastro/docs/commits/main/src/content/docs/en/guides/deploy/cloudflare.mdx) |
| `guides/view-transitions.mdx` | [source@`9049c78`](https://github.com/withastro/docs/blob/9049c78bf535b132d706a8b534f040c27958988d/src/content/docs/en/guides/view-transitions.mdx) | [`57d124a..9049c78`](https://i18n-tools.genteure.com/diff#withastro/docs/57d124a0451093fc8382f024f556dc9821783b7b..9049c78bf535b132d706a8b534f040c27958988d/src/content/docs/en/guides/view-transitions.mdx) | [blame@`9049c78`](https://github.com/withastro/docs/blame/9049c78bf535b132d706a8b534f040c27958988d/src/content/docs/en/guides/view-transitions.mdx) <br> [blame@`main`](https://github.com/withastro/docs/blame/main/src/content/docs/en/guides/view-transitions.mdx) <br> [history@`main`](https://github.com/withastro/docs/commits/main/src/content/docs/en/guides/view-transitions.mdx) |

### CODE REVIEW

File: src/content/docs/zh-cn/guides/view-transitions.mdx [hunk 1]
```diff
@@ -266,18 +266,70 @@ const myFade = {
 
 ### 触发导航
 
-你还可以使用 `navigate` 方法在通常不被 `<ViewTransitions />` 路由器监听的事件里触发客户端导航。该方法来自于 `astro:transitions/client` 模块，可以在任何客户端脚本或客户端组件中使用。
+你还可以使用 `navigate` 方法在通常不被 `<ViewTransitions />` 路由器监听的事件里触发客户端导航。该函数来自于 `astro:transitions/client` 模块，可以在任何客户端脚本和通过[客户端指令](/zh-cn/reference/directives-reference/#客户端指令)激活了的客户端组件中使用。
```
[the remark this answers is not in the document]
Comment (Genteure) [reply src/content/docs/zh-cn/guides/view-transitions.mdx:269]: 链接左右需要加空格吗？我觉得不加看起来效果更好一点。

在[链接](#)的[左右](#)添加空格。
在 [链接](#) 的 [左右](#) 添加空格

[the remark this answers is not in the document]
Comment (100gle) [reply src/content/docs/zh-cn/guides/view-transitions.mdx:269]: 也行

**Review by 100gle — CHANGES_REQUESTED** [review]
LGTM

**Review by 100gle — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🚀

**Pull request merged**